### PR TITLE
Creating a demo directory with a demo video and run script for CAPK

### PR DESCRIPTION
**Pull request by aburdenthehand**
Trialing an idea of including demo videos in the community repo ahead of KubeCon. This makes the demo broadly available for attendees to refer to during and after KubeCon, and also means that others will be able to present these demos at other confs, miniconfs, or meetups without having to re-create the environments and workflows each time. 

The CAPK demo was provided by @alicefr and I have included a demo script for it based on her presentation at KubeCon EU 22.

Signed-off-by: Andrew Burden <aburden@redhat.com>

### CODE REVIEW

**Comment by aburdenthehand** [comment]
/cc @alicefr 
Can you please give the script a review to ensure I have not introduced any inaccuracies?

File: demos/CAPK/demoscript_May-CAPK.txt [hunk 1]
```diff
@@ -0,0 +1,124 @@
+CAPK demo
+<Intro>
+In this demo, we are showing the Kubernetes Cluster API Provider KubeVirt, also known as CAPK, which is a defined way of creating and managing a Kubernetes cluster in a KubeVirt cluster... and by KubeVirt cluster I mean a Kubernetes cluster that has KubeVirt installed on it, which I want to differentiate from the VM cluster we will create within this cluster. 
+
+<Start>
+To begin, let's have a look at our KubeVirt cluster:
+We have two  nodes - one control plane and one worker node
+KubeVirt is already installed. 
+We can check the status of that and we see that it is deployed.
```
Comment (alicefr) [inline demos/CAPK/demoscript_May-CAPK.txt:9]: `We can check the status of the KubeVirt CR and we see that it is deployed`

File: demos/CAPK/demoscript_May-CAPK.txt [hunk 2]
```diff
@@ -0,0 +1,124 @@
+CAPK demo
+<Intro>
```
Comment (alicefr) [inline demos/CAPK/demoscript_May-CAPK.txt:2]: Maybe we could put a reference to the youtube video of KubeCon?

**Comment by aburdenthehand** [comment]
Thanks for the review @alicefr! I've made all changes as suggested.

**Comment by cwilkers** [comment]
Nice demo!
/lgtm

**Comment by alicefr** [comment]
/lgtm 
also from my side

**Comment by aburdenthehand** [comment]
/approve
Approving, now that this has two lgtm's

**Comment by aburdenthehand** [comment]
@cwilkers, looks like this needs your seal of approval

**Pull request merged**